conf/services/tax-you-paid.yml

```diff
@@ -2,9 +2,43 @@ serviceName: Check how much Income Tax you paid
 serviceHeaderName: Check how much Income Tax you paid
 serviceDescription: The Check how much Income Tax you paid service allows users that have received a P800 tax calculation to see why they have either overpaid or underpaid tax, and allows them to pay any tax owed.
 serviceDomain: www.tax.service.gov.uk
-serviceUrl: /tax-you-paid
-contactFrontendServiceId: tax-you-paid
-complianceStatus: noncompliant
+serviceUrl: /browse/tax/income-tax
```

Comment:
Is this url meant to change as it is different to the one in the Welsh file

Reply:
sorted